app/services/fdc_offer_broker.rb

```diff
@@ -33,8 +33,7 @@ def wholesale_product(product_id)
     production_flow = catalog_item("#{product_id}/AsPlannedProductionFlow")
 
     if production_flow
-      wholesale_product_id = production_flow.product
-      catalog_item(wholesale_product_id)
+      production_flow.product
```

Comment:
I have to admit I don't understand why we needed to load the catalog item before, or why we don't need to now..

Reply:
The `catalog_item` method looks up an object in the catalog, in this case it's returning the product. But with the updated context, the DFC importer knows that the URL is a product id and resolves that for us.